Mathlib/GroupTheory/Perm/Sign.lean

```diff
@@ -28,11 +32,13 @@ open Equiv Function Fintype Finset
 
 open BigOperators
 
-variable {α : Type u} {β : Type v}
+variable {α : Type u} [DecidableEq α] {β : Type v}
 
+/-
 -- An example on how to determine the order of an element of a finite group.
 example : orderOf (-1 : ℤˣ) = 2 :=
   orderOf_eq_prime (Int.units_sq _) (by decide)
+-/
```

Comment:
Should this be left commented out?

Reply:
The file does not import Mathlib.GroupTheory.OrderOfElement anymore, so the example does not compile.

Reply:
@AntoineChambert-Loir Do you think it makes sense to move it to a place where it does typecheck? If so, please mvoe it.

Reply:
I believe it was here because it has something to do with testing whether a permutation is even or odd… but the file where docs#orderOf is defined seems to be a good place.

Reply:
finally, I put it in Mathlib/GroupTheory/SpecificGroups/Alternating (otherwise, I needed to add an import…)